src/app/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-breadcrumbs',
+  templateUrl: './breadcrumbs.component.html',
+  styleUrls: ['./breadcrumbs.component.scss']
+})
+export class BreadcrumbsComponent implements OnInit {
+
+  constructor() { }
```

Comment:
just for aware - if you don't use constructor and other methods/hooks you can skip their declaration